lib/chewy/type/adapter/mongoid.rb

```diff
@@ -30,6 +30,41 @@ def import_scope(scope, options)
           end.all?
         end
 
+        def import_objects(collection, options)
```

Comment:
This is just a modified version of https://github.com/apolloio/chewy/blob/master/lib/chewy/type/adapter/orm.rb #import_objects